python/nano/src/bigdl/nano/deps/onnxruntime/pytorch/pytorch_onnxruntime_model.py

```diff
@@ -48,10 +48,12 @@ def __init__(self, model, input_sample=None, onnxruntime_session_options=None,
         onnx_path = model
         if isinstance(model, torch.nn.Module):
             # Typically, when model is fp32, we use this path
-            # TODO: expose ONNX export parameters to users
             export_to_onnx(model, input_sample=input_sample, onnx_path='tmp.onnx',
                            **export_kwargs)
             onnx_path = 'tmp.onnx'
+            # simplify model
+            from bigdl.nano.deps.onnxsim.onnxsim_api import onnx_simplify
+            onnx_simplify(onnx_path)
```

Comment:
does it always work? 

How about adding a flag to disable it?

Reply:
Yeah, if the conversion of onnx_simplify fails, onnx_path still save the original model.

I will add a flag to disable this.